client-app/components/molecules/select/select.vue

```diff
@@ -48,7 +48,7 @@
               @click="select(item)"
             >
               <slot name="item" v-bind="{ item, index, selected }">
-                {{ itemTextKey ? item[itemTextKey] : item }}
+                {{ itemTextKey && item ? item[itemTextKey] : item || "&nbsp;" }}
```

Comment:
Remove ` || "&nbsp;"`
This logic is at the mercy of the developer. To help `item` slot
Example in lines 43-46 `checkout-defaults.vue`